test/unit/Reflection/ReflectionPropertyTest.php

```diff
@@ -238,7 +238,7 @@ public function testGetDefaultProperty()
 
     public function testCannotClone()
     {
-        $classInfo = $this->reflector->reflect('\BetterReflectionTest\Fixture\ExampleClass');
+        $classInfo = $this->reflector->reflect('\Roave\BetterReflectionTest\Fixture\ExampleClass');
```

Comment:
We should use `::class` across the tests too. At least for the changes in this diff